### PR TITLE
SAAS-1128 - Added SALTO_LOG_MAX_JSON_MESSAGE_SIZE feature

### PR DESCRIPTION
**Pull request by amitgoldarb**

### CODE REVIEW

**Comment by amitgoldarb** [comment]
We would like to support integrations into different monitoring services (e.g datadog). Monitoring services typically have a predefined max message size. We would like to make the behaviour of such integrations more predictable and predefined by the logging package. For that reason we are adding a SALTO_LOG_MAX_JSON_MESSAGE_SIZE configurable parameter.

Release notes
Added SALTO_LOG_MAX_JSON_MESSAGE_SIZE option to logger configuration.

Added support to set predefined max message size in our logger.

File: packages/logging/README.md [hunk 1]
```diff
@@ -186,6 +186,24 @@ Add tags to all log messages. When configuring this variable via the CLI, make s
 
 Default: {} - Doesn't add any tags to log messages. 
 
+### `maxMessageSize: number | null`
+
+Environment variable:
+
+```bash
+SALTO_LOG_JSON_MAX_MESSAGE_SIZE=3k
```
Comment (royra) [inline packages/logging/README.md:194]: Should be 3KB according to the note below.

**Review by royra — APPROVED** [review]
Personal preference: I would remove the `bytes` lib dependency and simplify the env var to always be in bytes. Quantities in multiples of 1024 don't seem as useful to me - how many bytes is 200KB anyway?

Otherwise looks great!

File: packages/logging/README.md (hunk 1 above)
Comment (amitgoldarb) [reply packages/logging/README.md]: Updated after f2f discussion

**Pull request merged**